src/test/regress/expected/multi_truncate.out

```diff
@@ -432,8 +432,12 @@ CREATE TABLE dist(id int, ref_id int REFERENCES ref(id));
 INSERT INTO dist SELECT x,x FROM generate_series(1,10000) x;
 -- test that we do not cascade truncates to local referencing tables
 SELECT truncate_local_data_after_distributing_table('ref');
-ERROR:  cannot truncate a table referenced in a foreign key constraint by a local table
-DETAIL:  Table "dist" references "ref"
+NOTICE:  truncate cascades to table "dist"
```

Comment:
Interesting, the comment says we do not cascade, but now we cascade.

Is that working fine?

Reply:
a,h this works fine. The cascade is happening on the "shell" tables, which is fine and expected.